src/murmur/ServerUser.h

```diff
@@ -140,6 +140,8 @@ class ServerUser : public Connection, public User {
 	QList< int > qlCodecs;
 	bool bOpus;
 
+	QList< VoiceProtocol > supportedVoiceProtocols;
```

Comment:
```suggestion
	QVector< VoiceProtocol > supportedVoiceProtocols;
```
Why not `std::vector`?

Reply:
See above.